src/tcgwars/logic/impl/gen4/RisingRivals.groovy

```diff
@@ -247,15 +267,23 @@ public enum RisingRivals implements LogicCardInfo {
           resistance P, MINUS20
           pokeBody "Eerie Aura", {
             text "Put 1 damage counter on each of your opponent’s Pokémon that remains Asleep between turns."
-            delayedA {
+            before BEGIN_TURN, {
+              if(my.active.isSPC(ASLEEP)){
+                directDamage 10, self, SRC_ABILITY
+              }
```

Comment:
This Body says "each of **your opponent's Pokémon**"

Nothing about your Pokémon